src/lib/ical/utils.test.ts

```diff
@@ -77,35 +77,35 @@ test('wraps long text into lines broken at column 100 3 times', () => {
   assert(lines.at(-1).length === 1, 'where the last line is a single asterisk');
 });
 
-test('replaces HTML with external links', () => {
-  const actual = replaceHtmlWithExternalLinks(`<a href="https://example.com/">
+test('additional replaces HTML with external links tests', () => {
```

Comment:
Nit: Simplify the test name.
```suggestion
test('replaces link targets with "_blank" in HTML string', () => {
```